Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -3147,6 +3153,11 @@ bool RenderLayerBacking::isDirectlyCompositedImage() const
     if (m_owningLayer.hasVisibleBoxDecorationsOrBackground() || m_owningLayer.paintsWithFilters() || renderer().hasClip())
         return false;
 
+    // Fixed layers that allow detaching won't have a backing store,
+    // so using a directly composited image doesn't help (and has non-zero cost)
```

Comment:
The "directly composited image" comment confuses me. Didn't we lose that optimization with UI-side compositing?